.github/ISSUE_TEMPLATE/new-release.md

```diff
@@ -82,6 +82,7 @@ To keep track of spartacussampledata releases, we keep a `latest` branch on each
     - [ ] `npm run release:setup:with-changelog` (needed since `3.0.0-next.1`)
     - [ ] `npm run release:organization:with-changelog` (needed since `3.0.0-next.1`)
     - [ ] `npm run release:storefinder:with-changelog` (needed since `3.0.0-rc.0`)
+    - [ ] `npm run release:qualtrics:with-changelog` (needed since `3.0.0-rc.3`)
```

Comment:
not sure if it's 3.0.0-rc.3 milestone or 3.1.0

Reply:
I believe this is going to be released in 3.1, therefore I'd say: 3.1.0